test/integration/full-content/full-content.test.js

```diff
@@ -53,6 +53,9 @@ function normalizeParsedBlocks( blocks ) {
 		// Recurse to normalize inner blocks
 		block.innerBlocks = normalizeParsedBlocks( block.innerBlocks );
 
+		// Ignore since this isn't a part of the parser output
+		delete block.sourceMarkup;
```

Comment:
I think that's fine since we already have a dedicated snapshot for the source file, maybe we could also consider the same thing for originalContent.

Reply:
this sounds great. in fact I think we can go one step further and more positively talk about the attributes we want to check instead of removing the ones we don't. I'll post an update with what I mean.

Reply:
In [3a9ffa6](https://github.com/WordPress/gutenberg/pull/38460/commits/3a9ffa6e4f5e7772e8f91e3150dcefc403e0ffd0) I've adjusted this function but left `originalContent` for the same reason I didn't want to introduce the `sourceMarkup` - it turns this 30-line PR into a 700+ line PR. I wouldn't mind doing a follow-up PR though to update all the fixtures to remove the things we don't want and where that testing-methodology change could be reviewed by itself.